molo/core/templatetags/core_tags.py

```diff
@@ -165,6 +165,15 @@ def bannerpages(context):
     else:
         pages = []
 
+    if position >= 0:
+        banners = get_pages(context, pages, locale)
+        if position > (len(banners) - 1):
```

Comment:
I noticed an error in QA that will need us to do something similar in other tags